### ISSUE TITLE
wpt lint errors

### ISSUE THREAD

**Issue by sagudev** [gpuweb/cts#2628]
When running `wpt lint` on `out-wpt`:
```
ERROR:lint:webgpu/webgpu/shader/execution/memory_model/coherence.spec.js:467: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/shader/execution/expression/call/builtin/atomics/harness.js:49: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/shader/execution/expression/call/builtin/atomics/harness.js:117: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/web_platform/reftests/ref/canvas_complex-ref.html:24: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxSampledTexturesPerShaderStage.spec.js:35: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxSampledTexturesPerShaderStage.spec.js:63: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxSampledTexturesPerShaderStage.spec.js:97: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxSamplersPerShaderStage.spec.js:35: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxSamplersPerShaderStage.spec.js:63: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxSamplersPerShaderStage.spec.js:97: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:87: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:91: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:93: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:129: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:133: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:135: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:164: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:168: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:170: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:205: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:207: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:232: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:234: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:335: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:339: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:341: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:345: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/compute_pipeline/overrides.spec.js:349: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/labels.spec.js:233: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/operation/render_pipeline/overrides.spec.js:391: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxUniformBuffersPerShaderStage.spec.js:35: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxUniformBuffersPerShaderStage.spec.js:63: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/capability_checks/limits/maxUniformBuffersPerShaderStage.spec.js:97: Whitespace at EOL (TRAILING WHITESPACE)
ERROR:lint:webgpu/webgpu/api/validation/createTexture.spec.js:412: Test-file line starts with one or more tab characters (INDENT TABS)
ERROR:lint:webgpu/webgpu/api/validation/createTexture.spec.js:444: Test-file line starts with one or more tab characters (INDENT TABS)
ERROR:lint:webgpu/webgpu/idl/exposed.http.html: Visual test whose filename doesn't end in '-visual' (CONTENT-VISUAL)
ERROR:lint:webgpu/webgpu/idl/exposed.https.html: Visual test whose filename doesn't end in '-visual' (CONTENT-VISUAL)
INFO:lint:
INFO:lint:There were 37 errors (TRAILING WHITESPACE: 33 INDENT TABS: 2 CONTENT-VISUAL: 2)
INFO:lint:You must fix all errors; for details on how to fix them, see
INFO:lint:https://web-platform-tests.org/writing-tests/lint-tool.html
INFO:lint:
INFO:lint:However, instead of fixing a particular error, it's sometimes
INFO:lint:OK to add a line to the lint.ignore file in the root of the
INFO:lint:web-platform-tests directory to make the lint tool ignore it.
INFO:lint:
INFO:lint:For example, to make the lint tool ignore all 'CONTENT-VISUAL'
INFO:lint:errors in the webgpu/webgpu/idl/exposed.https.html file,
INFO:lint:you could add the following line to the lint.ignore file.
INFO:lint:
INFO:lint:CONTENT-VISUAL: webgpu/webgpu/idl/exposed.https.html
```

**Comment by sagudev** [comment]
Run on latest commit (https://github.com/gpuweb/cts/commit/0be3bde33c5abf01aef574403c888b8ec372453d) [reports](https://github.com/sagudev/cts/actions/runs/5077321557/jobs/9120445286#step:7:5):
```console
ERROR:lint:webgpu/common/util/navigator_gpu.js:63: Test-file line has a `console.*(...)` call (CONSOLE)
ERROR:lint:webgpu/webgpu/idl/exposed.http.html: Visual test whose filename doesn't end in '-visual' (CONTENT-VISUAL)
ERROR:lint:webgpu/webgpu/idl/exposed.https.html: Visual test whose filename doesn't end in '-visual' (CONTENT-VISUAL)
INFO:lint:
INFO:lint:There were 3 errors (CONSOLE: 1 CONTENT-VISUAL: 2)
INFO:lint:You must fix all errors; for details on how to fix them, see
INFO:lint:https://web-platform-tests.org/writing-tests/lint-tool.html
INFO:lint:
INFO:lint:However, instead of fixing a particular error, it's sometimes
INFO:lint:OK to add a line to the lint.ignore file in the root of the
INFO:lint:web-platform-tests directory to make the lint tool ignore it.
INFO:lint:
INFO:lint:For example, to make the lint tool ignore all 'CONTENT-VISUAL'
INFO:lint:errors in the webgpu/webgpu/idl/exposed.https.html file,
INFO:lint:you could add the following line to the lint.ignore file.
INFO:lint:
INFO:lint:CONTENT-VISUAL: webgpu/webgpu/idl/exposed.https.html
```

**Comment by kainino0x** [comment]
> ```
> ERROR:lint:webgpu/common/util/navigator_gpu.js:63: Test-file line has a `console.*(...)` call (CONSOLE)
> ```

Ah, I broke this one in fixing 

> ```
> ERROR:lint:webgpu/webgpu/idl/exposed.http.html: Visual test whose filename doesn't end in '-visual' (CONTENT-VISUAL)
> ERROR:lint:webgpu/webgpu/idl/exposed.https.html: Visual test whose filename doesn't end in '-visual' (CONTENT-VISUAL)
> ```

These are fixed by #2668.

**Comment by sagudev** [comment]
Closing as this is done.
Verified with https://github.com/gpuweb/cts/pull/2745
